.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 coverage/
+.nyc_output/
+src/
```

Comment:
Why gitignore this when it doesn't do anything in the codebase. What if you delete this folder?

Reply:
i've deleted the unnecessary folder